closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -261,13 +264,36 @@ static void printErrors(void)
     fflush(stderr);
 }
 
+/*
+ * We use a 8 digit map (AABBCCDD) to represent the version of openssl.
```

Comment:
The structure described in `NativeCrypto.java` used three digits for `DDD`.

Reply:
Hi @keithc-ca , may I know where the three digits are used for DDD? I didnt find it...

Reply:
See this in `NativeCrypto.java`:
```java
    public static final long OPENSSL_VERSION_1_0_0 = 0x1_00_00_000L;
```

Reply:
... and the multiplication by 1000 on line 286 (now 289) below.

Reply:
We use e.g. 10100000 for version 1.1.0 where AA is 10, BB is 10, CC is 00 and DD is 00. After changing to hexadecimal format (0x1_01_00_000L), A will be 1, BB will be 01, CC will be 00 and DDD will be 000, right?

Reply:
Edited.